src/_components/linkList/LinkList.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+
+import { Theme } from '../../_styles/theme';
+import Icon, { TIconName } from '../icon/Icon';
+import * as Styled from './style';
+
+type TLinkTypes = { href: string; onPress?: never } | { href?: never; onPress: () => void };
+
+export type TLinkListItem = {
+  iconColor: keyof Theme['colors'];
+  iconName: TIconName;
+  label: string;
+  labelColor: keyof Theme['colors'];
+} & TLinkTypes;
+
+type TProps = {
+  items: TLinkListItem[];
+};
+
+const LinkList: FC<TProps> = ({ items, ...props }) => {
+  function renderItem(item: TLinkListItem) {
+    return (
+      <Styled.LinkItem key={item.label}>
+        <Icon color={item.iconColor} name={item.iconName} />
+        {item.href ? (
```

Comment:
There's no added value for the if statement.. I think you can just pass in both the `href` and `onPress` props, since the typing matches the type of the `Button` props.

Reply:
no 😢